sycl/test-e2e/Matrix/joint_matrix_bfloat16.cpp

```
@@ -10,5 +10,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
+// XFAIL: gpu-intel-dg2
+
```

Same as above:
Adding XFAIL will make the code fail silently for the other supported combination.